main.go

```diff
@@ -70,61 +70,76 @@ func main() {
 	}
 }
 
-func check(e error) {
-	if e != nil {
-		panic(e)
-	}
+func get_kd_files(index int) (string, string) {
+	var keyfile = db + "/" + strconv.Itoa(index) + ".key"
+	var datafile = db + "/" + strconv.Itoa(index) + ".data"
+	return keyfile, datafile
 }
 
 func backup_etcd(ep string, dir string) error {
 	ctx := context.Background()
 	cli, err := clientv3.New(clientv3.Config{
 		Endpoints: []string{ep},
 	})
-	check(err)
+	if err != nil {
+		return err
```

Comment:
wrap the error with a description. also in other places

```suggestion
		return fmt.Errorf("failed to create etcd client: %w", err)
```